src/Parser/EnhancementShaman/CombatLogParser.js

```diff
@@ -232,6 +232,28 @@ class CombatLogParser extends MainCombatLogParser {
           </dfn>
         )}
       />,
+      hasHailstorm && (
+        <StatisticBox
+            icon={<SpellIcon id={SPELLS.FROSTBRAND.id} />}
+            value={`${formatPercentage(frostbrandUptime)} %`}
+            label={(
+              <dfn data-tip={`Goal should be 100% uptime`}>
+                Frostbrand Uptime
+              </dfn>
+            )}
+          />
+      ),
+      hasLandslide && (
+        <StatisticBox
+            icon={<SpellIcon id={SPELLS.LANDSLIDE_BUFF.id} />}
+            value={`${formatPercentage(landslideUptime)} %`}
+            label={(
+              <dfn data-tip={`Goal should be 100% uptime`}>
```

Comment:
Two things:
Tooltips can be added with the tooltip property of the StatisticBox, so labels shouldn't have a `<dfn>` element and can be simple strings.
This tooltip belongs in a suggestion and should not be shown here. Try to minimize the amount of tooltips so that they're not everywhere and users learn that if there's a tooltip it contains useful new information to them.